### PR TITLE
Dependencies update

### PR DESCRIPTION
**Pull request by jpsalo**
Updated dependencies that do not require upgrading React from 15 to 16. Most notable changes are:
 
- Webpack 1 → 4
- Babel 6 → 7
- old ESLint → `eslint-config-airbnb`

### CODE REVIEW

File: src/modules/unit/components/UnitBrowser.js [hunk 1]
```diff
@@ -17,25 +32,32 @@ import {
   getDefaultSportFilter,
 } from '../helpers';
 
-const ActionButton = ({action, icon, isActive}) =>
+const addressBarMarker = require('@assets/markers/location.svg');
```
Comment (jimppa) [inline src/modules/unit/components/UnitBrowser.js]: The icon is broken (at least for me locally), `import addressBarMarker from '@assets/markers/location.svg';` seems to work tho.

File: src/modules/unit/components/UnitFilters.js [hunk 2]
```diff
@@ -54,32 +64,33 @@ export class UnitFiltersComponent extends React.Component {
...
   }
 
   render() {
-    const {filters, t} = this.props;
-    const {expandedFilter} = this.state;
+    const { filters, t } = this.props;
+    const { expandedFilter } = this.state;
 
-    const FilterOptions = ({filter}: {filter: UnitFilterProps}) => (
+    const FilterOptions = ({ filter }: {filter: UnitFilterProps}) => (
```
Comment (jimppa) [inline src/modules/unit/components/UnitFilters.js]: I see that this was not changed now, but there is really no reason to create this component on every render, this should be separated to its own component outside render. Maybe add a fixme comment?

File: src/modules/unit/components/UnitsOnMap.js [hunk 3]
```diff
@@ -1,28 +1,31 @@
+/* eslint-disable react/prop-types */
```
Comment (jimppa) [inline src/modules/unit/components/UnitsOnMap.js]: There is a lot of these, maybe just disable this eslint rule?

File: package.json [hunk 4]
```diff
@@ -2,98 +2,103 @@
...
   "repository": {
-    "url": "git+https://github.com/digiaonline/outdoors-sports-map.git",
+    "url": "https://github.com/City-of-Helsinki/outdoors-sports-map.git",
     "type": "git"
   },
-  "author": "Christoffer Niska <christofferniska@gmail.com>",
-  "contributors": [
-    "Ville Vaarala <ville.l.vaarala@gmail.com>"
-  ],
+  "author": "Jukka-Pekka Salo <jpsalo@zoho.com>",
```
[the remark this answers is not in the document]
Comment (jpsalo) [reply package.json]: :sunglasses:

File: src/modules/unit/components/UnitsOnMap.js (hunk 3 above)
Comment (jpsalo) [reply src/modules/unit/components/UnitsOnMap.js]: There are following rationales to disable these locally:

1. The project uses Airbnb's ESLint where the intended behavior originates
2. React upgrade from 15 to 16 is planned but not started and the logic for PropTypes will go through some changes then if not earlier
3. The project has various conventions for syntax, most notably Flow which interferes with PropTypes. Flow will be changed to TypeScript and bring changes
4. The PR is rather large but it aims to change the actual React source code as little as possible (cf. 2 and 3)
5. The plan is to gradually migrate the source to conform Airbnb's "vanilla" ESLint when an individual file is subject to change in the upcomping feature implementation

File: src/modules/unit/components/UnitFilters.js (hunk 2 above)
Comment (jpsalo) [reply src/modules/unit/components/UnitFilters.js]: 7cb0c683a1d57371556a95f40250c99c43cd7f7c

**Pull request merged**